velox/type/parser/ParserUtil.h

```diff
@@ -36,4 +36,7 @@ std::pair<std::string, TypePtr> inferTypeWithSpaces(
     std::vector<std::string>& words,
     bool cannotHaveFieldName = false);
 
+/// Returns true only if 'str' contains digits.
```

Comment:
For single line comments, always use // (with a space separator before the comment).

https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md

Reply:
Fixed. Thanks.